src/openrct2-ui/interface/Window.cpp

```diff
@@ -87,8 +87,8 @@ static bool WindowFitsOnScreen(const ScreenCoordsXY& loc, int32_t width, int32_t
 }
 
 rct_window* WindowCreate(
-    const ScreenCoordsXY& screenCoords, int32_t width, int32_t height, rct_window_event_list* event_handlers,
-    rct_windowclass cls, uint16_t flags)
+    std::unique_ptr<rct_window> wp, rct_windowclass cls, const ScreenCoordsXY& pos, int32_t width, int32_t height,
```

Comment:
Shouldnt this be 
```suggestion
    std::unique_ptr<rct_window>&& wp, rct_windowclass cls, const ScreenCoordsXY& pos, int32_t width, int32_t height,
```

Reply:
I have done it, although not sure what difference it has made. It must have been moving it since it is a unique_ptr which can't be copied.

Reply:
It probably used the move constructor, but for clarity it should be there I think.